File: app/src/main/java/com/drdisagree/iconify/utils/RemotePrefProvider.java
package com.drdisagree.iconify.utils;

import com.crossbowffs.remotepreferences.RemotePreferenceFile;
import com.crossbowffs.remotepreferences.RemotePreferenceProvider;
import com.drdisagree.iconify.BuildConfig;

public class RemotePrefProvider extends RemotePreferenceProvider {
    public RemotePrefProvider() {
        super(BuildConfig.APPLICATION_ID, new RemotePreferenceFile[]{new RemotePreferenceFile(BuildConfig.APPLICATION_ID + "_xpreferences", true)});
    }
}

File: app/src/main/java/com/drdisagree/iconify/services/providers/RemotePrefProvider.kt
package com.drdisagree.iconify.services.providers

import com.crossbowffs.remotepreferences.RemotePreferenceFile
import com.crossbowffs.remotepreferences.RemotePreferenceProvider
import com.drdisagree.iconify.BuildConfig
import com.drdisagree.iconify.data.common.XposedConst.PREF_FILE_NAME

class RemotePrefProvider : RemotePreferenceProvider(
    BuildConfig.APPLICATION_ID,
    arrayOf(RemotePreferenceFile(PREF_FILE_NAME, true))
)